src/CoreWCF.WebHttp/src/CoreWCF/Channels/WebContentTypeMapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Channels
+{
+   public abstract class WebContentTypeMapper
+    {
```

Comment:
Indentation is wrong

Reply:
Addressed in 8722a911f6ec91bff5803c85cfe9082240d1c360.